integrations/clickup/main.py

```diff
@@ -0,0 +1,95 @@
+from enum import StrEnum
+from typing import Any
+from loguru import logger
+from port_ocean.context.ocean import ocean
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+from clickup.client import ClickupClient
+
+
+class ObjectKind(StrEnum):
+    TEAM = "team"
+    PROJECT = "project"
+    ISSUE = "issue"
+
+
+async def init_client() -> ClickupClient:
+    client = ClickupClient(
+        ocean.integration_config["clickup_base_url"],
+        ocean.integration_config["clickup_personal_token"],
+    )
+    return client
+
+
+async def setup_application() -> None:
+    app_host = ocean.integration_config.get("app_host")
+    if not app_host:
+        logger.warning(
+            "App host was not provided, skipping webhook creation. "
+            "Without setting up the webhook, you will have to manually initiate resync to get updates from ClickUp."
+        )
+        return
+    clickup_client = await init_client()  # Await the initialization of the client
+    await clickup_client.create_events_webhook(app_host)
+
+
+@ocean.on_resync(ObjectKind.TEAM)
+async def on_resync_teams(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    clickup_client = await init_client()
+    async for team in clickup_client.get_clickup_teams():
+        logger.info(f"Received team of length {len(team)}")
```

Comment:
what does that log means? `team` is an object, what a log would look like?

Reply:
Thank you for your feedback. The `team` variable is actually a list of objects, not a single object. The log statement is meant to provide information about the number of elements in this list. So, `len(team)` will give the number of team objects within the list. This is what the output will look like.

`Received team of length 5`

Reply:
I updated the variable name to reflect its more than one